### PR TITLE
fix(changedAttributes): initial value of property which are supposed to be null return a value when the record is in flight

### PR DESCRIPTION
**Pull request by VincentMolinie**

### CODE REVIEW

**Comment by knownasilya** [comment]
Shouldn't the test fail in all scenarios?

File: tests/unit/fragment_test.js [hunk 1]
```diff
@@ -201,6 +201,51 @@ module('unit - `MF.Fragment`', function (hooks) {
...
+      null,
+      'old fragment is indicated in the diff object'
+    );
+    assert.deepEqual(
+      newName,
+      { first: 'Rob', last: 'Stark', prefixes: [] },
+      'new fragment is indicated in the diff object'
+    );
+
+    person._internalModel.adapterWillCommit();
```
Comment (VincentMolinie) [inline tests/unit/fragment_test.js:233]: I don't know if there is a better way to pass the record in a willCommit state

**Comment by VincentMolinie** [comment]
@knownasilya yeah it was not ready sorry 😉

**Comment by VincentMolinie** [comment]
@knownasilya it's ready 😉

**Comment by VincentMolinie** [comment]
Can anyone look at it please 👼  ?

**Comment by knownasilya** [comment]
I'll have a look tomorrow, or Monday.

**Review by knownasilya — APPROVED** [review]
Nice

**Comment by knownasilya** [comment]
Released

**Pull request merged**